### DISCUSSION TITLE
Latest EULA / license

### DISCUSSION THREAD

**Discussion by JDT3004** [jgraph/drawio-desktop#1897] [General]
Can anybody provide a link to the latest EULA / license for the desktop version. Security are requesting it for my organisation. Many thanks

**Comment by davidjgraph** [comment]
https://github.com/jgraph/drawio-desktop/blob/dev/LICENSE

**Reply by JDT3004** [reply]
> https://github.com/jgraph/drawio-desktop/blob/dev/LICENSE

Thank you very much
